std_msgs/Header header
uint8 index
int16 hs_var1
int16 hs_var2
